client/src/components/ComponentInfo/ComponentDetails/index.jsx

```diff
@@ -26,6 +30,4 @@ const ComponentDetails = ({ subHeading, codeState, setCodeState, children }) =>
 };
 
 const EnhancedComponentDetails = withCodeToggle(ComponentDetails);
```

Comment:
Another question here. Can you explain why you are doing this(hoc)? Why not have the codeState and toggleState inside ComponentDetails.

Reply:
Yes, I could've managed the state inside the `ComponentDetails` itself. But I wanted to experiment with **HOC's** that's the only reason I implemented the HOC here